aos/aos_milestones/IMpg/check_impg_run.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>Running the notebook seems to show that this has multiple sets of Zernikes for each CCD.</p><p>In that case, where are the wavefront results for each sensor? Are those what the OFC directly uses?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Uhm.. this is a good point. This wavefront errors are the "raw" values computed by the wep. Because these values can be combined in different ways (with different weighing methods, for instance), I thought it would be good to publish the "raw" values.. But I think you are right that later we also want to publish the values used by the OFC.. I think we should probably add different events in the MTAOS for that. One for raw wavefront errors and one for the combined ones. </p><p> </p>